test/test_location.js

```diff
@@ -369,6 +392,44 @@ describe('states.location', function() {
             .run();
         });
 
+        it('should display skip option when skip text not null',
+        function() {
+            tester.data.opts.options_per_page = 1;
+            tester.data.opts.skip_text = 'Skip';
+            return tester
+            .inputs("Friend Street")
```

Comment:
I read that wrong, I read intended instead on indented. Thanks :) ready for review once again :)